packages/libra-rpc-graphql/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@shopify/libra-rpc-graphql",
   "version": "2.5.2",
   "license": "MIT",
-  "description": "GraphQL utilities for connecting to the Libra network via the JSON-RPC API.",
+  "description": "GraphQL utilities for creating an Apollo link that connects to the Libra network via the JSON-RPC API.",
```

Comment:
improving the description as this package is really focused on building an apollo link.